src/pyodide/internal/workers.py

```diff
@@ -1,6 +1,7 @@
 # This module defines a Workers API for Python. It is similar to the API provided by
 # JS Workers, but with changes and additions to be more idiomatic to the Python
 # programming language.
+import datetime
 import functools
 import http.client
 import json
```

Comment:
Not 100% connected to this PR but we should consider some way to prevent people from doing `from workers import js` or `from workers import datetime`. There are two possible approaches:
1. `import datetime as _datetime`
2. move the implementation to `_workers.py` and then import just the public things into `workers.py`.

I favor option 2.

Reply:
Will do in separate PR.